adept/_vlasov1d/solvers/vector_field.py

```diff
@@ -155,14 +155,23 @@ def __init__(self, cfg: Dict):
         else:
             raise NotImplementedError
         self.fp = fokker_planck.Collisions(cfg=cfg)
+        self.vlasov_dfdt = cfg["diagnostics"]["diag-vlasov-dfdt"]
+        self.fp_dfdt = cfg["diagnostics"]["diag-fp-dfdt"]
```

Comment:
I support adding module-level fields like this so we don't have to reference `cfg` in the `call` methods.